Documentation/k8s-pre-reqs.md

```diff
@@ -18,7 +18,9 @@ you can quickly set one up using [Minikube](#minikube), [Kubeadm](#kubeadm) or [
 
 ## Minimum Version
 
-Kubernetes v1.11 or higher is supported by Rook.
+Kubernetes **v1.11** or higher is supported by Rook.
+
+**Important** If you are using K8s 1.15 or older, you will need to create a different version of the Rook CRDs. Use the common.yaml found in the [k8s-1.15](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/k8s-1.15) subfolder of the example manifests.
```

Comment:
In ceph-common-issues.md, how about adding a troubleshooting section that shows an error if they try to use the v1 resources on k8s 1.15 or older? Then the topic can point them to the old crds.

Reply:
Also on ceph-quickstard.md at the end of the section "Deploy the Rook Operator", how about adding a note that if they're on k8s 1.15 or older they need to create the older crds?

Reply:
Done.